controller/src/controller.rs

```diff
@@ -197,7 +205,10 @@ impl<T: BottlerocketShadowClient> BrupopController<T> {
                         self.progress_node(brs).await;
                     }
                 }
-            } else {
+            }
+
+            // Bring one more node each time if the active nodes size is less than MAX_CONCURRENT_UPDATE setting.
+            if active_set_size < get_max_concurrent_update()? {
```

Comment:
This logic looks like we need call `get_max_concurrent_update` frequently (once per loop). I wonder if we can cache `max_concurry_update` value when we generate `controller`, so we don't have to re-call this function so many times.

Reply:
This was by design so any update for the environment variable could quickly fetch by the controller